arraycontext/fake_numpy.py

```diff
@@ -145,30 +145,30 @@ def _get_fake_numpy_linalg_namespace(self):
     _c_to_numpy_arc_functions = {c_name: numpy_name
             for numpy_name, c_name in _numpy_to_c_arc_functions.items()}
 
-    #def __getattr__(self, name):
-        #def loopy_implemented_elwise_func(*args):
-            #actx = self._array_context
-            #prg = _get_scalar_func_loopy_program(actx,
-                    #c_name, nargs=len(args), naxes=len(args[0].shape))
-            #outputs = actx.call_loopy(prg,
-                   # **{"inp%d" % i: arg for i, arg in enumerate(args)})
-            #return outputs["out"]
-
-        #if name in self._c_to_numpy_arc_functions:
-            #from warnings import warn
-            #warn(f"'{name}' in ArrayContext.np is deprecated. "
-                    #"Use '{c_to_numpy_arc_functions[name]}' as in numpy. "
-                    #"The old name will stop working in 2021.",
-                    #DeprecationWarning, stacklevel=3)
-
-        # normalize to C names anyway
-        #c_name = self._numpy_to_c_arc_functions.get(name, name)
+    def __getattr__(self, name):
+        def loopy_implemented_elwise_func(*args):
+            actx = self._array_context
+            prg = _get_scalar_func_loopy_program(actx,
+                    c_name, nargs=len(args), naxes=len(args[0].shape))
+            outputs = actx.call_loopy(prg,
+                    **{"inp%d" % i: arg for i, arg in enumerate(args)})
+            return outputs["out"]
+
+        if name in self._c_to_numpy_arc_functions:
+            from warnings import warn
+            warn(f"'{name}' in ArrayContext.np is deprecated. "
+                    "Use '{c_to_numpy_arc_functions[name]}' as in numpy. "
+                    "The old name will stop working in 2021.",
+                     DeprecationWarning, stacklevel=3)
+
+        #normalize to C names anyway
+        c_name = self._numpy_to_c_arc_functions.get(name, name)
```

Comment:
This should be set to fire.

Reply:
Removed arraycontext.fake_numpy.__getattr__ ->  Implemented impl.pyopencl.fake_numpy.__getattr__